stack2stack.py

```
@@ -123,26 +126,69 @@ def migrate_images():
                 if not path.isfile(i.name):
-                    with open(i.name, 'wb') as f:
-                        for chunk in old_cloud_glance_client.images.data(i.id):
-                            f.write(chunk)
-                    f.close()
+                    print 'New Cloud: image - %s does not yet exist' % i.name
+                    print ''
+                    print 'Local Filesytem: Downloading temp image to local disk'
+                    print '                 Preparing to transfer %s to New Cloud' % i.name
+                    print ''
+                    try:
+                        with open(i.name, 'wb') as f:
+                            for chunk in old_cloud_glance_client.images.data(i.id):
+                                f.write(chunk)
+                        f.close()
+                    except glanceclient.exc.HTTPInternalServerError:
+                        print ''
+                        print 'Error: There was a problem uploading this Image - %s - ID: %s - This was caused by an HTTPInternalServerError at this time. You may want to investigate further' % (i.name, i.id)
+                        print ''
+                        continue
                 if not is_public:
                     old_cloud_glance_client.images.update(i.id, is_public=False)
 
                 new_owner_id = new_cloud_keystone_client.tenants.find(name=old_cloud_keystone_client.tenants.find(id=i.owner).name, description=old_cloud_keystone_client.tenants.find(id=i.owner).description).id
-                j = new_cloud_glance_client.images.create(id=i.id, name=i.name, is_public=is_public, disk_format = i.disk_format, container_format = i.container_format, owner = new_owner_id)
+                try:
+                    j = new_cloud_glance_client.images.create(id=i.id, name=i.name, is_public=is_public, disk_format = i.disk_format, container_format = i.container_format, owner = new_owner_id)
+                    except:
```

if you can change this exception to be
except: glanceclient.ext.HTTPConflict:

Just add a new commit to the branch and push it to this merge request. Also check that it's a HTTPConflict exception that is being thrown when this particular use case happens (I'm just taking an educated guess). Globally swallowing all exceptions can lead to problems.